src/landing/LandingPage.js

```diff
@@ -12,38 +12,33 @@ const LandingPage = () => {
   const [form, setForm] = useState('Login');
   const [usernameState, setUsernameState] = useState(null);
   const [passwordState, setPasswordState] = useState(null);
+  const [loginResult, setLoginResult] = useState(null);
+  const [registerResult, setRegisterResult] = useState(null);
 
   const [getUsers, {
-    data: queryData, error: queryError, loading: queryLoading,
+    data: queryData, error: queryError,
   }] = useLazyQuery(GET_USERS);
 
   useEffect((() => {
     if (queryData && usernameState && passwordState) {
-      console.log(queryData);
       if (queryData.users.find((user) => user.username === usernameState
           && user.password === passwordState)) {
-        alert(`Login successful for ${usernameState}`);
+        setLoginResult('success');
```

Comment:
should import and use `loginStates.credentialsError` and `loginStates.success` and `loginStates.serverError` here as well instead of the strings. The point of the comment is that the constant is used in both files which means they will never get out of sync/ strings are not repeated.